templates/firefox-ios.xliff

```diff
@@ -10,7 +10,7 @@
         <note>Privacy - Camera Usage Description</note>
       </trans-unit>
       <trans-unit id="NSFaceIDUsageDescription" xml:space="preserve">
```

Comment:
Soooooooo...... this is a problem. Unlike 99.9% of our other strings, this comes from a `info.plist` whose identifiers are set by the OS/Apple. Unlike our other strings, we don't have control over these identifiers. If we change this identifier, the OS doesn't pick up the existence of this string and won't use it, because it's a designated system prompt. And there's no way around this, unfortunately.

So, what I think has to happen is that you might have to delete this string in Pontoon and then we have to get it retranslated fully. I'm open to other suggestions, as long as none of them involve changing the identifier. lol

Reply:
I see.
Is it possible at all to delete it on your side for just a moment, then I run a string import and expose that import to Pontoon, and then you add it back?
I think that might be an option, looking into other options as well
(note: I'm sick so going to take the rest of the day off, I may only be able to address this Monday)

Reply:
Actually, nevermind, I was just reminded about [this](https://github.com/mozilla-l10n/firefoxios-l10n#string-updates): "As a consequence, the default update removes translations if:
The source text was changed."
So, we're all good for this.

Reply:
ok. Well, lemme know and I can delete it on my side and we can re-run then if it doesn't work.

Reply:
Hope you feel better!!!! :)